style.css

```diff
@@ -27,28 +27,33 @@ h1 {
   font-weight: 700;
   width: 100%;
   font-size: 19.85vw;
+  font-size: 20.85vw;
   color: rgb(255, 254, 254);
   line-height: 1;
   font-family: Helvetica, sans-serif;
+  font-family: "Josefin Sans", sans-serif;
```

Comment:
The duplicate `font-size` and `font-family` definitions here and below should be merged into single ones.